docs/src/modules/components/AppLayoutDocsFooter.js

```diff
@@ -310,6 +349,24 @@ export default function AppLayoutDocsFooter() {
     setSnackbarOpen(false);
   };
 
+  React.useEffect(() => {
+    const eventListener = (event) => {
+      const feedbackHash = event.target.getAttribute('data-feedback-hash');
+      if (feedbackHash) {
+        const section = sectionOptions.find((item) => item.hash === feedbackHash) || EMPTY_SECTION;
+        setCommentOpen(true);
+        setCommentedSection(section);
+        if (inputRef.current) {
+          inputRef.current.focus();
+        }
```

Comment:
It can occurs without being a problem

If comment section is closed, cliking on the button will open it. At that time the input is not yet rendered so the ref is still `undefined`, but the focus will bet set on the input thanks to the `handleEntered` callback

If the comment section is already open, this section will set the focus and

Reply:
Ok, I see.